block_iagora.php

```diff
@@ -47,35 +47,43 @@ public function get_content() {
         }
 
         $this->content = new stdClass();
-        $this->content->footer = '';
-
         $iframeurl = isset($this->config->iframeurl) ? $this->config->iframeurl : '';
-
         if (empty($iframeurl)) {
             $this->content->text = get_string('noiframeurl', 'block_iagora');
         } else {
             $this->content->text = '<iframe src="' . $iframeurl . '" width="100%" height="400px" frameborder="0"></iframe>';
```

Comment:
```suggestion
            $this->content->text = $this->generate_chat_content($iframeurl);
```
We don't need the iframe anymore, right?

Reply:
okay!